cmake/sysbuild/sdp.cmake

```diff
@@ -8,6 +8,10 @@ function(sdp_apply_snippets snippet)
   sysbuild_cache_set(VAR sdp_SNIPPET APPEND REMOVE_DUPLICATES ${snippet})
 endfunction()
 
+function(sdp_apply_flpr_fault_timeout timeout)
+  sysbuild_cache_set(VAR ${DEFAULT_IMAGE}_CONFIG_SDP_FAULT_TIMEOUT APPEND REMOVE_DUPLICATES ${timeout})
```

Comment:
not how we set Kconfigs from sysbuild, `set_config_*` functions exist though `set_config_int` is waiting for upmerge but this should be using `set_property(TARGET ${image} APPEND_STRING PROPERTY CONFIG "${setting}=${value}\n")`